### DISCUSSION TITLE
Offer alternative mirror for registry.suse.com

### DISCUSSION THREAD

**Discussion by prabhu** [SUSE/bci#23] [General]
We are getting 500 errors for registry.suse.com. We are also seeing images where nodejs20, npm20, gcc12 etc have simply vanished indicating possible issues with the package feeds. Do you offer any alternative registry mirrors? Also what is the best way to access the sha256 hashes for the images in your registry?

**Comment by prabhu** [comment]
Any ETA for the registry issues? Or could you share any alternative support channels to track the issue?

**Comment by msmeissn** [comment]
this is not monitored on the weekend.

**Reply by prabhu** [reply]
Any exceptions? It is unfortunately serving the old SP4 images with vulnerabilities for all unsuspecting users.

**Reply by msmeissn** [reply]
i emailed the respective team, lets see if someone can pick it up

**Comment by msmeissn** [comment]
bug was fixed, registry pieces are currently being repopulated. this can take some hours :/

**Reply by prabhu** [reply]
Thank you so much! Is there any plans to publish an RCA and actions to prevent this (SP5 - SP4 downgrade) from happening in the future?

**Reply by prabhu** [reply]
I noticed that a number of images were restored yesterday. Thank you so much! I have a suspicion that the hashes published on the website do not match the output from docker inspect command. I found this with a couple of openjdk-devel images. Happy to retest later this week once things are fully restored.

**Reply by dirkmueller** [reply]
We have written up a RCA and will publish this shortly. currently trying to determine the most appropriate place.

**Reply by dirkmueller** [reply]
Published here under https://github.com/SUSE/bci/discussions/25 for now. We will find a more permanent spot and also consider the input to provide a redundancy mirror.
